src/ipc-utils.ts

```diff
@@ -38,6 +38,8 @@ export interface IpcEvents {
 		},
 	];
 	'site-context-menu-action': [ { action: string; siteId: string } ];
+	'studio-file-upload-paused': [ { error: string; selectedSiteId: string; remoteSiteId: number } ];
+	'studio-file-upload-resumed': [ { selectedSiteId: string; remoteSiteId: number } ];
```

Comment:
```suggestion
	'sync-upload-paused': [ { error: string; selectedSiteId: string; remoteSiteId: number } ];
	'sync-upload-resumed': [ { selectedSiteId: string; remoteSiteId: number } ];
```
Nit, but prefixing with `studio-` seems superfluous. Specifying that it's related to sync operations might be useful, though.

Reply:
It makes sense to keep it related to sync operations. Added in [b2a7760](https://github.com/Automattic/studio/pull/2212/commits/b2a7760069214b4f06be8b548b8aa72b3a0ca05e).